.github/workflows/release.yml

```diff
@@ -0,0 +1,12 @@
+name: CI build
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up node
+        uses: actions/setup-node@v1
+      - name: Install dependencies with yarn
```

Comment:
please properly name the step - here dependencies are not installed by yarn, but by npm!